release-squad.md

```diff
@@ -0,0 +1,94 @@
+- Release Team: https://make.wordpress.org/core/handbook/about/release-cycle/wordpress-release-team-and-focus-leads/ -- some key sections:
+	- Shared Responsibilities: https://make.wordpress.org/core/handbook/about/release-cycle/wordpress-release-team-and-focus-leads/#shared-responsibilities
+	- Test Responsibilities: https://make.wordpress.org/core/handbook/about/release-cycle/wordpress-release-team-and-focus-leads/#test
+
+Additionally, now is a good time to bookmark key Make WordPress resources, which grow and evolve as the release unfolds. Look for this release's "Development Cycle", "Planning", and "Roadmap" posts for a good start. Here are some examples from 6.3:
+
+- Release Hub: https://make.wordpress.org/core/6-3/
+- Planning: https://make.wordpress.org/core/2023/05/18/wordpress-6-3-planning-roundup/
+- Roadmap: https://make.wordpress.org/core/2023/05/18/roadmap-to-6-3/
+
+Be on the lookout for posts about the release, features, etc -- consume as much as you can.
+
+And finally, make sure to join the appropriate release leads channel in [Make WordPress Slack](https://make.wordpress.org/chat/) to coordinate with other release squad teams, and to stay in the loop on latest developments! 👍
+
+## Call for Testing Post
+The official release Call for Testing (CfT) post helps align the testing community around the key features and enhancements planned for the release. An example is the [CfT for WordPress 6.1](https://make.wordpress.org/test/2022/09/21/help-test-wordpress-6-1/).
+
+### Timing
+In recent releases, the CfT has shipped just before the Beta 1 release post goes out on [wordpress.org/news](http://wordpress.org/news), where the release post typically includes a link to the CfT. The thought behind this timing is to allow testing to begin right away, and for bugs to be uncovered early in the cycle. However, there isn't often much urgency at this point, and most testing typically ramps up around RC1.
+
+The CfT is one of the first major tasks for the Test release team. It should be noted that during the beta cycle, there are usually outstanding questions as to what *exactly* will officially make it into the release 😅 With this timeline and ambiguity, it can feel frantic putting it together!
+
+So to be clear: **It’s OKAY for the CfT to be edited/updated during the release cycle, or for follow-up CfTs to be published afterwards.** The post's change log is used to track updates, and major updates may even warrant detailed separate posts in Make/Core. If you’ve followed past releases, you'll notice there are always adjustments made because we're all human, and things fall through the cracks. But most features are typically settled by RC1.
+
+### Post Format
+The format of the CfT was adapted from the [WordPress 5.9 CfT](https://make.wordpress.org/test/2021/11/30/help-test-wordpress-5-9-features/), where some of the post is boilerplate, but the majority of content is specific to new release features.
```

Comment:
Yes, that is what I suggested [here](https://github.com/WordPress/test-handbook/pull/71/files#r2468240641) to create a pattern